### PR TITLE
GVRMaterial, GVRShaderData, GVRShaderManager docs

### PR DESCRIPTION
**Pull request by NolaDonato**
documentation only - no code changes

GearVRF DCO signed off by: Nola Donato nola.donato@samsung.com

### CODE REVIEW

**Comment by liaxim** [comment]
Why is GVRBitmapTexture still called that and not GVRBitmapImage?

**Comment by NolaDonato** [comment]
Because I did not get around to changing all the names. This would require changing demos too. If you think we have time to make name changes before we release 4.0, I am open to it.

**Comment by NolaDonato** [comment]
I am done with this PR now. I had to fix some formatting in the light docs too.

File: GVRf/Framework/framework/src/main/java/org/gearvrf/GVRMaterial.java [hunk 1]
```diff
@@ -47,17 +42,17 @@
  * 
  * <p>
  * The simplest way to create a {@link GVRMaterial} is to call the
- * {@linkplain GVRMaterial#GVRMaterial(GVRContext) constructor that takes only a
- * GVRContext.} Then you just {@link GVRMaterial#setMainTexture(GVRTexture)
- * setMainTexture()} and you're ready to draw with the default shader, which is
+ * {@linkplain GVRMaterial#GVRMaterial(GVRContext)} constructor that takes only a
+ * GVRContext. Then you just {@link GVRMaterial#setTexture(String, GVRTexture)}
+ * and you're ready to draw with the default shader, which is
  * called 'unlit' because it simply drapes the texture over the mesh, without
```
Comment (liaxim) [inline GVRf/Framework/framework/src/main/java/org/gearvrf/GVRMaterial.java]: The default shader has not been called 'unlit' for a very long time.

**Pull request merged**